src/js/hca-rjsf/routes.jsx

```diff
@@ -0,0 +1,18 @@
+import { createRoutes as createFormRoutes } from '../common/schemaform/helpers';
+import reducer from './reducer';
+import formConfig from './config/form';
+import HealthCareApp from './HealthCareApp.jsx';
+
+export default function createRoutes(store) {
```

Comment:
We can simplify this file. The `createRoutes` function was created so that we could replace the Redux store whenever a different form was loaded. Since this app has just one form, it's not necessary. You can do:

```js
const route = {
  path: '/',
  component: HealthCareApp,
  indexRoute: { onEnter: (nextState, replace) => replace('/introduction') },
  childRoutes: createFormRoutes(formConfig)
};

export default route;
```